Hearthstone Deck Tracker/GameEventHandler.cs

```diff
@@ -868,7 +868,20 @@ private async void HandleConstructedStart()
 					{
 						var cards = Core.Game.Player.PlayerEntities.Where(x => x.IsInHand && !x.Info.Created).Select(x => x.Card.DbfIf);
 						var opponentClass = Core.Game.Opponent.PlayerEntities.FirstOrDefault(x => x.IsHero && x.IsInPlay)?.Card.CardClass ?? CardClass.INVALID;
-						Core.Overlay.ShowMulliganPanel(shortId, cards.ToArray(), opponentClass);
+						var goingFirst = Core.Game.Player.GoingFirst;
+
+						var isWild = _game.CurrentFormat == Format.Wild;
+						var isClassic = _game.CurrentFormat == Format.Classic;
+
+						var playerStarLevel = 0;
+						if (_game.MatchInfo != null)
+						{
+							var localPlayer = _game.MatchInfo.LocalPlayer;
+							var playerInfo = isClassic ? localPlayer.Classic : isWild ? localPlayer.Wild : localPlayer.Standard;
+							playerStarLevel = playerInfo?.LeagueId >= 5 ? playerInfo?.StarLevel ?? 0 : 0;
```

Comment:
I believe you can drop the LeagueId check here? That was only relevant when they introduced the new rank system. Everything should be >= 5 nowadays.